ProjFS.Mac/PrjFSKextLogDaemon/PrjFSKextLogDaemon.cpp

```diff
@@ -204,6 +221,7 @@ static bool TryFetchAndLogKextHealthData(io_connect_t connection)
     IOReturn ret = IOConnectCallStructMethod(connection, LogSelector_FetchVnodeCacheHealth, nullptr, 0, &healthData, &out_size);
     if (ret == kIOReturnUnsupported)
     {
+        WriteJsonToMessageListener("{\"ErrorMessage\":\"IOConnectCallStructMethod failed for LogSelector_FetchVnodeCacheHealth, kIOReturnUnsupported \"}");
```

Comment:
Did you want this to be a 2 field message? Later you have an error message with “ErrorMessage” and “ret” fields.